__tests__/app.test.js

```diff
@@ -396,3 +397,25 @@ describe("Task 10 GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (topic query)", () => {
+  test("GET 200. return array of articles with provided topic", () => {
+    return request(app)
+      .get("/api/articles?topics=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
```

Comment:
You'd ideally want to check the other keys are there as well as making sure the topic is mitch

Reply:
This feedback hasn't been addressed yet. This would be a stronger test if you made sure you're checking all of the properties, not just the topic.

You can use a flexible matcher such as toMatchObject inside your for each in order to do this